apps/ws/src/Game.ts

```diff
@@ -124,19 +125,24 @@ export class Game {
       return;
     }
 
+    let WhitePlayer = users.find((user) => user.id === this.player1UserId);
```

Comment:
also here maybe const whitePlayer etc...

Reply:
changed those variables to constants